README.md

```diff
@@ -139,15 +139,14 @@ Note: The asset hierarchy builder is currently in beta, and has not been suffici
 on production data.
 
 The `.option("type", "assethierarchy")` lets you write new asset hierarchies, or update existing ones,
-using the Spark Data Source. It currently supports writing a single root asset and any number of children.
-The asset hierarchy builder can ingest entire hierarchies, as long as all nodes are connected to the root asset
+using the Spark Data Source.
+The asset hierarchy builder can ingest entire hierarchies of nodes connected
 through the `externalId`/`parentExternalId` relationship. If input contains an update to data that already exists,
 i.e there's a match on `externalId` and there's a change to one of the other fields, the asset will be updated.
 There's also an option to delete assets from CDF that are not referenced in the input data.
 
 ### Requirements
-- The source data must contain a single root, denoted by setting its `parentExternalId` to the empty string `""`.
-- All assets, except root, must be connected to another asset in the input data by `parentExternalId`.
+- The root is denoted by setting its `parentExternalId` to the empty string `""`.
```

Comment:
If we now support multiple roots, maybe change this to

- Root assets are denoted by setting their `parentExternalId` to the empty string `""`.